trigger/streamtester/timer.go

```diff
@@ -148,25 +148,29 @@ func (t *Trigger) start(handler trigger.Handler, settings *HandlerSettings, emit
 		if settings.Block {
 			data, err := ReadCsv(settings.FilePath)
 			if err != nil {
+				t.logger.Debug("Error while reading csv.", err)
 				return
 			}
 
 			triggerData = prepareOnceData(data, settings.Header)
 		} else {
 			dataTemp, err := ReadCsvInterval(settings.FilePath, emitInfo)
 			if err != nil {
+				t.logger.Debug("Error while reading csv.", err)
 				return
 			}
 			triggerData = prepareRepeatingData(dataTemp, emitInfo, settings.Header)
 
 			emitInfo.Count = emitInfo.Count + 1
 			//triggerData.Data = dataTemp
 		}
-		
-		t.logger.Debug("Data passed to Handler..",triggerData )
+
+		t.logger.Debug("Data passed to Handler..", triggerData.Data)
```

Comment:
This seems less helpful than just triggerData, consider:

t.logger.Debug("Trigger data: %#v", triggerData)